src/main/webapp/comments.html

```diff
@@ -12,8 +12,9 @@
       </header>
       <main>
         <div class="topnav">
+
           <a href="college-info.html">School Info</a>
-          <a class="active">Student Reviews</a>
+          <a class="active" href="comments.html">Student Reviews</a>
```

Comment:
same here. The comments page is actually useless now since comments and info are all on the college-info page so we can delete it soon